app/controllers/ingredient_controller.rb

```diff
@@ -0,0 +1,47 @@
+class IngredientController < ApplicationController
+  def new
+    if params[:name]
+      i = Ingredient.create(:name => params[:name])
```

Comment:
plz always use a good naming for variable  never and ever use a one-letter variable, if you return one month, year later you donc remember what `i`